Documentation/kubernetes-on-aws-launch.md

```diff
@@ -0,0 +1,67 @@
+# Launch your Kubernetes cluster on AWS
+
+This is the [third step of running Kubernetes on AWS][aws-step-1]. We're ready to launch the Kubernetes cluster.
+
+## Create the CloudFormation stack
+
+Now for the exciting part, create your cluster:
+
+```sh
+$ kube-aws up
+```
+
+**NOTE**: It can take some time after `kube-aws up` completes before the cluster is available. When the cluster is first being launched, it must download all container images for the cluster components (Kubernetes, dns, heapster, etc). Depending on the speed of your connection, it can take a few minutes before the Kubernetes api-server is available.
+
+## Configure DNS
+
+If you configured Route 53 settings in your configuration above via `createRecordSet`, a host record has already been created for you.
+
+Otherwise, navigate to the DNS registrar hosting the zone for the provided external DNS name. Ensure a single A record exists, routing the value of `externalDNSName` defined in `cluster.yaml` to the externally-accessible IP of the master node instance.
+
+You may use `kube-aws status` to get this value after cluster creation, if necessary. This command can take a while.
```

Comment:
s/may use/can invoke/
s/this value/the cluster API IP address/
s/This command can take a while.//